pandas/sparse/panel.py

```diff
@@ -65,7 +65,7 @@ class SparsePanel(Panel):
     _typ = 'panel'
     _subtyp = 'sparse_panel'
 
-    def __init__(self, frames, items=None, major_axis=None, minor_axis=None,
+    def __init__(self, frames={}, items=None, major_axis=None, minor_axis=None,
```

Comment:
you don't want to put a mutable item in the constructor. rather, make the default ``None``. Then check if its ``None`` and assign in the code. Otherwise, you will get aliasing (e.g. the same dict will be assigned to *multiple* series)